<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Image_Proccesing</class>
 <widget class="QWidget" name="Image_Proccesing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image_Proccesing</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Folder path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="path">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>521</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_preview">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>511</width>
     <height>391</height>
    </rect>
   </property>
   <property name="text">
    <string>image_preview</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>500</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Crop Values:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>580</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="width_start">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>530</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="width_end">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>530</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="height_end">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>570</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="height_start">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>570</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="crop_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>610</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
